server/datastore/mysql/vulnerabilities_test.go

```diff
@@ -35,6 +35,7 @@ func TestVulnerabilities(t *testing.T) {
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
+			t.Helper()
```

Comment:
This looks interesting. Do you have an example how this helps?

Reply:
`t.Helper()` is really useful to mark a test helper function (a func resused in a test func, or across test funcs that has assertions in it), so when a test error occurs, the test results point you to where the function is called rather than where in the helper function it failed (although that is provided as well).  This is actually a bad placement of one, so i'm taking it out.